ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -129,6 +130,8 @@ data NodeKernelArgs m addrNTN addrNTC blk = NodeKernelArgs {
       tracers                 :: Tracers m (ConnectionId addrNTN) addrNTC blk
     , registry                :: ResourceRegistry m
     , cfg                     :: TopLevelConfig blk
+    , tolerableClockSkew      :: ClockSkew
```

Comment:
@amesgen I'd appreciate your thoughts on whether these two fields are the right ones to add.

- It's kind of weird to have both `SystemTime` and `BlockchainTime`.

- Maybe we want to cordon these off into a `chainSyncArgs` hierarchical field?

Reply:
a

Reply:
Hmm, yeah, I like that better.

Reply:
Addressed :+1: